### PR TITLE
chore: update the outdated README.md

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue #

### CODE REVIEW

**Comment by derekbit** [comment]
@coderabbitai pause

**Review by c3y1huang — COMMENTED** [review]
Should we also include sections for build status, contribution, license, etc., similar to those in the longhorn-engine and other repos?

**Comment by derekbit** [comment]
> Should we also include sections for build status, contribution, license, etc., similar to those in the longhorn-engine and other repos?

All are updated. Thank you.

File: README.md [hunk 1]
```diff
@@ -1,219 +1,26 @@
...
-	libibverbs-dev libfuse-dev libiscsi-dev \
-	zlib1g-dev libfdt-dev libpcap0.8-dev \
-	libncurses-dev libcunit1-dev \
-	build-essential nasm autoconf libtool automake
-```
+Please check [the main repo](https://github.com/longhorn/longhorn#community) for the contributing guide.
 
-3. Configure and build the SPDK fork:
+## License
+Copyright (c) 2014-2024 The Longhorn Authors
```
Comment (c3y1huang) [inline README.md:14]:
```suggestion
Copyright (c) 2022-2024 The Longhorn Authors
```

**Review by c3y1huang — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (derekbit) [reply README.md]: Fixed. Thanks.

**Review by c3y1huang — APPROVED** [review]

**Pull request merged**